plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/OpenShiftCoreMessages.properties

```diff
@@ -12,13 +12,14 @@ DebugOnOpenshift=Debug on Openshift
 ProfileOnOpenshift=Profile on Openshift
 RunOnOpenshift=Run on Openshift
 
-OCBinaryLocationIncompatibleErrorMessage=OpenShift client version 1.1.1 or higher is required to avoid rsync issues. You may <a>download</a> and/or <a>configure</a> a newer version.
-OCBinaryLocationDontExistsErrorMessage=OpenShift client ({0}) does not exists. You may <a>download</a> and/or <a>configure</a> a new one.
-NoOCBinaryLocationErrorMessage=OpenShift client not configured. You may <a>download</a> and/or <a>configure</a> it.
+OCBinaryLocationIncompatibleErrorMessage=OpenShift client 'oc' version 1.1.1 or higher is required to avoid rsync issues. You may <a>download</a> and/or <a>configure</a> a newer version.
+OCBinaryLinuxRSyncPermErrorMessage=OpenShift client 'oc' version 3.11 is required to avoid permission errors when rsync'ing from a Linux client. You may <a>download</a> and/or <a>configure</a> a newer version.
```

Comment:
`rsync'ing` is it an intentional single quote?

Reply:
@bdshadow didnt think at how single quotes were avoinding param substitution. Luckily there are none behind that, but no, it's not intentional. Fixing.